### PR TITLE
minor project search improvements

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1108

### Summary of Changes

* fix storybook
* tweak colors

### CODE REVIEW

File: src/components/ProjectSearch/TextSearch.css [hunk 1]
```diff
@@ -48,12 +54,18 @@
...
+  color: white;
+}
+
 .project-text-search .file-result .arrow {
   margin: 2px 0 2px 0;
 }
 
 .project-text-search .file-result.focused {
-  background-color: #eeeeee;
+  background-color: var(--theme-selection-background);
```
Comment (jasonLaster) [inline src/components/ProjectSearch/TextSearch.css]: these colors are a small step in the right direction

File: src/components/stories/TextSearch.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
-import React, { DOM as dom } from "react";
+import React, { DOM as dom, createFactory } from "react";
 import { storiesOf } from "@storybook/react";
 import _TextSearch from "../ProjectSearch/TextSearch";
 const TextSearch = React.createFactory(_TextSearch);
+
+import _Shortcuts from "./helpers/shortcuts";
+const Shortcuts = createFactory(_Shortcuts);
```
Comment (codehag) [inline src/components/stories/TextSearch.js]: Are we still using this pattern?

File: src/components/stories/TextSearch.js [hunk 3]
```diff
@@ -15,37 +19,40 @@ if (typeof window == "object") {
...
       {
-        className: `search-bar ${themeClass}`,
-        dir,
+        className: "",
         style: {
-          width: "100vw",
-          "align-self": "center"
+          width: "calc(100vw - 100px)",
+          height: "calc(100vh - 100px)",
+          margin: "auto",
```
Comment (codehag) [inline src/components/stories/TextSearch.js]: my gut instinct is to move styles that are not calculated to a class, but it was like this before so...

**Comment by codehag** [comment]
we have a failing test: 

```
*** End BrowserChrome Test Results ***

The following tests failed:

5751 INFO TEST-UNEXPECTED-FAIL | devtools/client/debugger/new/test/mochitest/browser_dbg-search-project.js | undefined assertion name - Got project, expected null

Stack trace:

    chrome://mochikit/content/browser-test.js:test_is:911

    chrome://mochitests/content/browser/devtools/client/debugger/new/test/mochitest/browser_dbg-search-project.js:null:56

    Tester_execTest@chrome://mochikit/content/browser-test.js:735:9

    Tester.prototype.nextTest</<@chrome://mochikit/content/browser-test.js:655:7

    SimpleTest.waitForFocus/waitForFocusInner/focusedOrLoaded/<@chrome://mochikit/content/tests/SimpleTest/SimpleTest.js:791:59

Buffered messages finished

SUITE-END | took 164s


./bin/run-mochitests-docker returned exit code 1
```

File: src/components/stories/TextSearch.js (hunk 2 above)
Comment (jasonLaster) [inline src/components/stories/TextSearch.js]: I suppose we should JSX the stories too, but meh

File: src/components/stories/TextSearch.js (hunk 3 above)
Comment (jasonLaster) [inline src/components/stories/TextSearch.js]: yeah - it's a silly story, which is where I'm not sure

**Comment by jasonLaster** [comment]
the test is fixed by the other PR

**Pull request merged**